scanner/mappers/mappers.go

```diff
@@ -120,6 +121,9 @@ func ToClairCoreIndexReport(contents *v4.Contents) (*claircore.IndexReport, erro
 		}
 	}
 	return &claircore.IndexReport{
+		State:         r.GetState(),
```

Comment:
This part was missing, which became apparent once the Matcher wrapped the call to the remote indexer inside of `getClairIndexReport`